### ISSUE TITLE
[Tab Extended]: QA: On applying text spacing, no changes is observed in the tooltip content.

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9235]
### Description

Page Element: Tooltip content

Issue Description:
On applying text spacing, no changes is observed in the tooltip content.

Expected:On applying text spacing, page should get appended accordingly. Spaces between the text should change and should be presented as per the standard.

User Impact:
If there is a loss of content or functionality When users override author specified text spacing to improve their reading experience, it affects users with low vision / user suffering from dyslexia

### Component(s) impacted

Tab extended
Tab extended with media

### Browser

Firefox

### Carbon for IBM.com version

v1.37.0-rc.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component canary: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component staging: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
